sdks/python/apache_beam/runners/portability/flink_runner.py

```diff
@@ -43,7 +43,8 @@ def default_job_server(self, options):
 class FlinkRunnerOptions(pipeline_options.PipelineOptions):
   @classmethod
   def _add_argparse_args(cls, parser):
-    parser.add_argument('--flink_master', default='[local]')
+    parser.add_argument('--flink_master', '--flink_master_url',
+                        default='[local]')
```

Comment:
The default for Java is `[auto]`. Should we change the Python SDK to match?

Reply:
Good point! We should.